### ISSUE TITLE
setTimeout and setInterval not working?

### ISSUE THREAD

**Issue by junedev** [patriksimek/vm2#34]
I just saw that setTimeout and setInterval are not working inside the VM. I am using version 3.0.0.

Example 1:
```
const {NodeVM} = require('vm2');
let vm = new NodeVM();
vm.run('setTimeout(function(){console.log(12345)}, 10000);');
```
Instead of printing 12345 after 10s this prints it immediately.

Example 2:
```
const {NodeVM} = require('vm2');
let vm = new NodeVM();
vm.run('let counter = 0; function foo(){console.log(counter++);} setInterval(foo, 5000);');
```
Instead of increasing and printing the counter every 5s it does it what seems like every millisecond.

I didn't see any comment in the documentation that those functions do not work. Is this a bug or did I overlook something?

**Comment by donothingloop** [comment]
Seems like I'm affected by the same bug.

**Comment by patriksimek** [comment]
Fixed in 3.0.1
